src/debug_utils.cc

```diff
@@ -100,16 +100,14 @@ class PosixSymbolDebuggingContext final : public NativeSymbolDebuggingContext {
 
 std::unique_ptr<NativeSymbolDebuggingContext>
 NativeSymbolDebuggingContext::New() {
-  return std::unique_ptr<NativeSymbolDebuggingContext>(
-      new PosixSymbolDebuggingContext());
+  return std::make_unique<PosixSymbolDebuggingContext>();
 }
 
 #else  // HAVE_EXECINFO_H
 
 std::unique_ptr<NativeSymbolDebuggingContext>
 NativeSymbolDebuggingContext::New() {
-  return std::unique_ptr<NativeSymbolDebuggingContext>(
-      new NativeSymbolDebuggingContext());
+  return std::unique_ptr<NativeSymbolDebuggingContext>();
```

Comment:
```suggestion
  return std::make_unique<NativeSymbolDebuggingContext>();
```

?

Reply:
std::unique_ptr<T>() sets the internal pointer to nullptr, whereas std::make_unique<T>() actually allocates memory

Reply:
I think @TimothyGu is right – it should allocate memory here, like it previously did, no?

Reply:
https://en.cppreference.com/w/cpp/memory/unique_ptr/unique_ptr

1) Constructs a std::unique_ptr that owns nothing. Value-initializes the stored pointer and the stored deleter. Requires that Deleter is DefaultConstructible and that construction does not throw an exception.

Reply:
@Yuhanun ... yes, what @TimothyGu and @addaleax are pointing out here is that the original code *is* allocating memory...

Original:
```c++
  return std::unique_ptr<NativeSymbolDebuggingContext>(
      new NativeSymbolDebuggingContext());
```

@TimothyGu's suggested edit is correct.

Reply:
Oh, did I change it to std::unique_ptr instead of std::make_unique? Okay ill commit suggestion